enginetest/enginetests.go

```diff
@@ -1025,7 +1025,6 @@ func TestInsertIntoErrors(t *testing.T, harness Harness) {
 
 func TestBrokenInsertScripts(t *testing.T, harness Harness) {
 	for _, script := range queries.InsertBrokenScripts {
-		t.Skip()
```

Comment:
If these are all fixed now, move the queries to `InsertScripts` and and delete the `TestBrokenInsertScripts` test functions.